### ISSUE TITLE
Not an error but....

### ISSUE THREAD

**Issue by copasetical1** [soleon/Percentage#22]
When BPI starts, I am greeted with the basic popup window 

![image](https://github.com/user-attachments/assets/ba273b5a-80fe-4407-913f-2e93a86394d1)

When I go to settings and change to start minimised, The setting is not saving, and is turned off each restart:

![BPI_Reset](https://github.com/user-attachments/assets/d20c68e9-c196-44c6-be46-c46eac598081)

Interestingly enough IIRC I have always started minimised. All I can do at this point is blame Windows Update.

**Comment by soleon** [comment]
Hi @copasetical1 thanks for reporting this issue.

This seems to be a common issue that a small portion of users are experiencing which I have yet to be able to pinpoint the root cause.

Were you updated from the previous 1.3 version with the old design to this newer 2.0 version with the new fluent design?

If you did, can you check if you have the following folder:
```
C:\Users\<your user name>\AppData\Local\Soleon_Innovations\Percentage.Wpf.exe_Url_<unique ID>\1.0.0.0
```

If you do, try the following:
1. Delete the `Soleon_Innovations` folder.
2. Uninstall the app.
3. Reinstall it from Microsoft Store.
4. Try changing any settings.
5. Right click the tray icon and exit the app.
6. Start the app again see if the changes are remembered.

**Comment by copasetical1** [comment]
Oh I was not. I think I need to get the newer version? Tbh, I downloaded
this from the Windows Store and I cannot seem to find it now. Ao maybe
that's the problem?  If you removed it that's fine I'm guessing the new
version is on GitHub (which I'm new to but I can probably figure it out).

**Comment by soleon** [comment]
@copasetical1 the better way to install the app is via the Microsoft Store. It has the latest version and I'm actively maintaining it again.

**Comment by copasetical1** [comment]
Yes, sorry, that's what I was saying. When I go to the store it's no longer
listed.  I will try later and see if the store ITSELF needs
updating...maybe the software selections are out of date.
I still get to blame windows itself, LOL

**Comment by soleon** [comment]
@copasetical1 if you can't find the app in Microsoft Store, here is the direct link to it: https://apps.microsoft.com/detail/9PCKT2B7DZMW.
